src/main/java/com/microsoft/aad/msal4j/ITenantProfile.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.util.Map;
+
+/**
+ * Interface representing a single tenant profile. ITenantProfiles are made available through the
+ * {@link IAccount#getTenantProfiles()} method of an Account
+ *
+ */
+public interface ITenantProfile {
+
+    /**
+     * This value corresponds to the 'oid' key of an ID token
+     *
+     * @return String local OID
+     */
+    String getId();
+
+    /**
+     * This value corresponds to the 'realm' key of an ID token
+     *
+     * @return String local tenant ID
```

Comment:
not sure we should use "local" here

Reply:
Good catch, now that we don't have the home/non-home tenants distinction there's no need to refer to these sorts of things as 'local'. Will remove in the next commit.